templates/download/risc-v/tab-2.html

```diff
@@ -2,6 +2,16 @@
 	<div class="row u-vertically-center u-no-padding">
 		<div class="col-6">
 			<h2>Download Ubuntu Server</h2>
+			<h3 class="p-heading--4">Ubuntu Server 22.10</h3>
+			<div class="p-notification--information">
+				<div class="p-notification__content">
+					<h5 class="p-notification__title">Note:</h5>
```

Comment:
This note looks different from the same notes in the other tabs.
<img width="587" alt="image" src="https://user-images.githubusercontent.com/57550290/196201823-2647c74b-6243-4181-9148-5f03e2b98b4e.png">
<img width="569" alt="image" src="https://user-images.githubusercontent.com/57550290/196201887-b01df395-59c3-46e6-af86-894b7fe22c4f.png">